tests/ci_tests/code_formatter/crewai/test_crewai_agents.py

```diff
@@ -0,0 +1,310 @@
+"""Tests for coffee_maker/code_formatter/crewai/agents.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent
+
+from coffee_maker.code_formatter.crewai import agents
+from coffee_maker.code_formatter.crewai.tools import PostSuggestionToolLangAI
+
+
+class TestLLMConfiguration:
+    """Tests for LLM configuration"""
+
+    def test_llm_exists(self):
+        """Test that llm object is initialized"""
+        assert agents.llm is not None
+
+    def test_llm_initialization_error(self, monkeypatch):
+        """Test that missing API keys raise a runtime error."""
```

Comment:
Added a docstring to describe the class's purpose.
```suggestion
class TestCreateCodeFormatterAgents:
```